packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -375,26 +375,27 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
       onPermitOthersVideo: canForbidOthersVideo ? muteAllHandlers.onPermitOthersVideo : undefined,
       meetingMediaAccess
     };
+    return {};
   }, [
-    canForbidOthersAudio,
-    onForbidAudio,
-    onPermitAudio,
-    muteAllHandlers.onForbidOthersAudio,
-    muteAllHandlers.onPermitOthersAudio,
-    muteAllHandlers.onForbidOthersVideo,
-    muteAllHandlers.onPermitOthersVideo,
-    canForbidOthersVideo,
-    onForbidVideo,
-    onPermitVideo,
-    meetingMediaAccess
+    /* @conditional-compile-remove(media-access) */ canForbidOthersAudio,
+    /* @conditional-compile-remove(media-access) */ onForbidAudio,
+    /* @conditional-compile-remove(media-access) */ onPermitAudio,
+    /* @conditional-compile-remove(media-access) */ muteAllHandlers.onForbidOthersAudio,
+    /* @conditional-compile-remove(media-access) */ muteAllHandlers.onPermitOthersAudio,
+    /* @conditional-compile-remove(media-access) */ muteAllHandlers.onForbidOthersVideo,
+    /* @conditional-compile-remove(media-access) */ muteAllHandlers.onPermitOthersVideo,
+    /* @conditional-compile-remove(media-access) */ canForbidOthersVideo,
+    /* @conditional-compile-remove(media-access) */ onForbidVideo,
+    /* @conditional-compile-remove(media-access) */ onPermitVideo,
+    /* @conditional-compile-remove(media-access) */ meetingMediaAccess
   ]);
 
   const { isPeoplePaneOpen, openPeoplePane, closePeoplePane } = usePeoplePane({
     ...peoplePaneProps,
     ...spotlightPeoplePaneProps,
     ...onMuteParticipantPeoplePaneProps,
     ...pinPeoplePaneProps,
-    /* @conditional-compile-remove(media-access) */ ...onToggleParticipantMicPeoplePaneProps
+    ...onToggleParticipantMicPeoplePaneProps
```

Comment:
Kinda curious why the original cc didn't work but either way looks fine to me

Reply:
Separators don't work with cc, hence the need to return an empty object in the props.